From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Tue, 11 Oct 2022 00:42:44 +0900
Subject: udev: add one more assertion

Follow-up for 6209bbbd4b1c9ed2886028ab2ee3df0a7d0e2494.

(cherry picked from commit f4a4493eb57bffec091c8f0d4bb846850e544aa4)
---
 src/udev/udev-builtin-path_id.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/udev/udev-builtin-path_id.c b/src/udev/udev-builtin-path_id.c
index 1084eb2..4cdc8d1 100644
--- a/src/udev/udev-builtin-path_id.c
+++ b/src/udev/udev-builtin-path_id.c
@@ -556,6 +556,7 @@ static int find_real_nvme_parent(sd_device *dev, sd_device **ret) {
          *   /sys/devices/pci0000:00/0000:00:1c.4/0000:3c:00.0/nvme/nvme0 */
 
         assert(dev);
+        assert(ret);
 
         r = sd_device_get_sysname(dev, &sysname);
         if (r < 0)
